capstone-project/src/main/angular-webapp/src/styles.css

```diff
@@ -0,0 +1,12 @@
+/* You can add global styles to this file, and also import other style files */
+#map-container {
+    height: 95vh;
+}
+
+h1 {
+    text-align: center;
+  }
+
+h2 {
+    text-align: center;
+}
```

Comment:
Similarly, why not define this in app.component.css? the relevant h1/h2 tags appear in app.component.html